cosmos/cosmos_test.go

```diff
@@ -4,3 +4,129 @@
 // LICENSE file in the root directory of this source tree.
 
 package cosmos
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/resources"
+	"github.com/Azure/azure-sdk-for-go/services/cosmos-db/mongodb"
+	"github.com/globalsign/mgo/bson"
+)
+
+var (
+	accountName = "cosmos-db-account-samples-" + helpers.GetRandomLetterSequence(10)
+)
+
+func TestMain(m *testing.M) {
+	flag.StringVar(&accountName, "cosmosDBAccountName", accountName, "Provide a name for the CosmosDB account to be created")
+
+	err := helpers.ParseArgs()
+	if err != nil {
+		log.Fatalln("failed to parse args")
+	}
+	os.Exit(m.Run())
+}
+
+func ExampleCosmosDBOperations() {
+	accountName := strings.ToLower(accountName)
+	helpers.SetResourceGroupName("CosmosDBOperations")
+	ctx := context.Background()
+	defer resources.Cleanup(ctx)
+	_, err := resources.CreateGroup(ctx, helpers.ResourceGroupName())
```

Comment:
I noticed that in most cases the result returned from the API isn't captured and I'm curious as to why.

Reply:
Well, it is not really needed (I have nothing to do with the `ResourceGroup` struct here)